packages/graphiql/.gitignore

```diff
@@ -9,7 +9,7 @@ analyzer.html
 /graphiql.*js.map
 /graphiql.*css
 /graphiql.*css.map
-example/yarn.lock
+example/pnpm.lock
```

Comment:
```suggestion
example/pnpm-lock.yaml
```

Reply:
can you tell I did a cheap bulk find replace and it's a mess ? haha